app/models/googletrans_client.py

```
@@ -0,0 +1,41 @@
+    Google Translate API (googletrans) を用いてテキストを翻訳するクライアントクラス。
+
+    Attributes:
+        client: googletransのTranslatorインスタンス
+    """
+    async def async_translate(self, text: str, src: str, dest: str):
+        """
+        Google Translate API (googletrans) を用いてテキストを翻訳します。
+
+        Args:
+            text (str): 翻訳対象のテキスト。
+            src (str): 翻訳元の言語コード（例: 'en', 'ja'）。
```

Ruff (0.14.0)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)